src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -255,7 +253,7 @@ public Schema fetchTableSchema(String tableName, FailureCollector collector) {
     Schema schema = null;
     try {
       schema = fetchTableSchema(tableName);
-    } catch (OAuthProblemException | OAuthSystemException | RuntimeException e) {
+    } catch (OAuthProblemException | OAuthSystemException | RuntimeException | IOException e) {
```

Comment:
We can simplify the catch by using catch(Exception), can you check and update ?

Reply:
@MrRahulSharma In general it is recommended to catch specific exception in lower stack and catch general exception (the ones that are remaining) at caller level, which I guess Sai was doing. However, if you think it is still recommended please go ahead.